CHANGELOG.md

```diff
@@ -2,7 +2,69 @@
 
 ## [Unreleased](https://github.com/CosmWasm/cw-plus/tree/HEAD)
 
-[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.10.3...HEAD)
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.11.0...HEAD)
+
+## [v0.11.0](https://github.com/CosmWasm/cw-plus/tree/v0.11.0) (2021-11-22)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/untagged-c9db5b9c4f390bdfe5f3...HEAD)
```

Comment:
```suggestion
[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.10.3...v0.11.0)
```
and it will work when the 0.11 version tag is created.

Reply:
Missed that, thank you.